ethvm/main.cpp

```diff
@@ -181,29 +178,35 @@ int main(int argc, char** argv)
 			mode = Mode::OutputOnly;
 		else if (arg == "trace")
 			mode = Mode::Trace;
-		else
-			incoming = arg;
+		else if (arg == "--input" && i + 1 < argc)
+			transactionDataString = argv[++i];
+		else if (arg == "--code" && i + 1 < argc)
+			contractCodeString = argv[++i];
 	}
 
 	VMFactory::setKind(vmKind);
 
-	bytes code;
-	if (incoming == "--" || incoming.empty())
```

Comment:
`--input` and filepath arguments are independent, see https://github.com/ethereum/libethereum/pull/278/files#diff-95f10d2c41ff73513a9d323e1d0fa531L184. We just need to decide what to do if both are present. I suggest print an error end exit.